srcalc/src/main/java/gov/va/med/srcalc/web/controller/DisplayResultsController.java

```diff
@@ -0,0 +1,78 @@
+package gov.va.med.srcalc.web.controller;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import gov.va.med.srcalc.domain.workflow.CalculationWorkflow;
+import gov.va.med.srcalc.vista.VistaPatientDao;
+import gov.va.med.srcalc.web.view.Views;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.dao.RecoverableDataAccessException;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class DisplayResultsController
+{
+	private final VistaPatientDao fPatientDao;
+    
+    @Inject
+    public DisplayResultsController(final VistaPatientDao patientDao)
+    {
+        fPatientDao = patientDao;
+    }
+    
+    @RequestMapping(value = "/displayResults", method = RequestMethod.GET)
+    public String displayResults(
+            final HttpSession session,
+            final Model model)
+    {
+        // Get the CalculationWorkflow from the session.
+        final CalculationWorkflow workflow = CalculationWorkflowSupplier.getWorkflowFromSession(session);
+        
+        model.addAttribute("calculation", workflow.getCalculation());
+        return Views.DISPLAY_RESULTS;
+    }
+    
+    @RequestMapping(
+    		value="/signCalculation",
+    		method = RequestMethod.POST,
+            produces = "application/json")
+    @ResponseBody
+    public HashMap<String, String> signCalculation(final HttpSession session, 
+    		@RequestParam(value = "eSig") final String electronicSignature,
+    		final Model model)
+    {
+    	// Build the note body and submit the RPC
+    	String resultString;
+    	try
+    	{
+    		resultString  = fPatientDao.saveRiskCalculationNote(
```

Comment:
We're still jumping right from the Web layer to the DAO layer. Even though the code will be very simple, I think we should add a service-layer method for consistency.